etna/core/mixins.py

```diff
@@ -32,11 +32,11 @@ def __repr__(self):
             if param.kind == param.VAR_POSITIONAL:
                 continue
             elif param.kind == param.VAR_KEYWORD:
```

Comment:
Let's create some basic tests on working with `BaseMixin`. The methods to functionality to check is:
- `repr`
  - Add test on dummy class with private attribute and public property
- `to_dict`
  - We already have file `test_to_dict`, let's add new tests here
  - Add test on dummy class with private attribute and public property
- `set_params`
  - We already have file `test_set_params`, let's add new tests here
  - Add test on dummy class with private attribute and public property